src/ngraph/runtime/interpreter/int_executable.hpp

```diff
@@ -1769,7 +1767,9 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
         case OP_TYPEID::ShapeOf:
         case OP_TYPEID::ShapeOf_v3:
         case OP_TYPEID::Softmax:
-        case OP_TYPEID::Subtract: NGRAPH_CHECK(false, "Op not handled by evaluator method:", node);
+        case OP_TYPEID::Squeeze:
+        case OP_TYPEID::Subtract:
+        case OP_TYPEID::Unsqueeze: NGRAPH_CHECK(false, "Op not handled by evaluator method:", node);
```

Comment:
I know the code is not yours, but that NGRAPH_CHECK does nothing.

Reply:
How does it do nothing?